src/components/Loader/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleList } from './PeopleList';
+import { getPeople } from '../../../api';
+import { Person } from '../../../types';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
```

Comment:
Consider handling the `setPeople` and `setIsLoading` inside the `.then` and `.finally` with caution. If the component unmounts before the promises resolve, this can lead to a React state update on an unmounted component, which will throw a warning. To avoid this, you can use an isMounted pattern or check if the component is still mounted before setting state.